os/include/unistd.h

```diff
@@ -51,6 +51,7 @@
  ****************************************************************************/
 /**
  * @defgroup UNISTD_KERNEL UNISTD
+ * @brief Provides APIs for Standard symbolic constants and Types
```

Comment:
I'm not sure but how about applying the option '1' for consistent presentations? :)